.github/ISSUE_TEMPLATE/01-new-components.yml

```diff
@@ -0,0 +1,22 @@
+name: New component
+description: Document a component that needs to be added to the GOV.UK Publishing Design Guide
+title: "Document [Enter the name of the component]"
+labels: ["component", "new documentation"]
+projects: ["alphagov/99"]
```

Comment:
should this not be the design guide?

Reply:
If you're referring to `projects` this refers to the [Community Board](https://github.com/orgs/alphagov/projects/99/).

The way the syntax works is `[name of user or organization]`/`[board number, which can be found in the board's URL]`